graphdatascience/tests/integration/conftest.py

```diff
@@ -126,11 +126,11 @@ def clean_up(gds: GraphDataScience) -> Generator[None, None, None]:
     for pipeline_name in res["pipelineName"]:
         gds.pipeline.get(pipeline_name).drop(failIfMissing=True)
 
-    res = gds.beta.model.list()
+    res = gds.model.list()
     for model_info in res["modelInfo"]:
         model = gds.model.get(model_info["modelName"])
         if model.stored():
-            gds.alpha.model.delete(model)
+            gds.model.delete(model)
```

Comment:
for these to work you need to copy them from the `model_beta_proc_runner`to the `model_proc_runner` (keep the beta part for compatability)